src/PKSim.Core/Services/ExpressionProfileUpdater.cs

```diff
@@ -41,36 +58,57 @@ public class ExpressionProfileUpdater : IExpressionProfileUpdater
       {
          return _individualExpressionTask.RenameMolecule(expressionProfile.Molecule, expressionProfile.MoleculeName, expressionProfile.Individual);
       }
 
-      public void SynchronizeExpressionProfile(ISimulationSubject simulationSubject, ExpressionProfile expressionProfile)
+      private void synchronizeExpressionProfiles(IndividualMolecule sourceMolecule, ISimulationSubject sourceSimulationSubject, IndividualMolecule targetMolecule, ISimulationSubject targetSimulationSubject, bool updateParameterOriginId)
       {
-         _lazyLoadTask.Load(simulationSubject);
-         var molecule = simulationSubject.MoleculeByName(expressionProfile.MoleculeName);
-         
+
          //Global settings for molecule
-         updateGlobalMoleculeSettings(molecule, expressionProfile, simulationSubject);
+         updateGlobalMoleculeSettings(sourceMolecule, targetMolecule, targetSimulationSubject);
 
          //All molecule parameters
-         updateMoleculeParameters(molecule, expressionProfile, simulationSubject);
-         
+         updateMoleculeParameters(sourceMolecule, sourceSimulationSubject, targetMolecule, targetSimulationSubject, updateParameterOriginId);
+
          //Molecule containers
-         updateTransporterDirections(molecule, expressionProfile, simulationSubject);
+         updateTransporterDirections(sourceMolecule, sourceSimulationSubject, targetMolecule, targetSimulationSubject);
+      }
+
+      public void SynchroniseSimulationSubjectWithExpressionProfile(ISimulationSubject simulationSubject, ExpressionProfile expressionProfile)
+      {
+         _lazyLoadTask.Load(simulationSubject);
+         var moleculeInIndividual = simulationSubject.MoleculeByName(expressionProfile.MoleculeName);
+
+         // ExpressionProfile => SimulationSubject, we want to make sure that the parameters in simulation subject are linked to their expression profile origin parameters
+         synchronizeExpressionProfiles(expressionProfile.Molecule, expressionProfile.Individual, moleculeInIndividual, simulationSubject, updateParameterOriginId:true);
+      }
+
+      public void SynchronizeExpressionProfileWithSimulationSubject(ExpressionProfile expressionProfile, ISimulationSubject simulationSubject)
+      {
+         var moleculeInIndividual = simulationSubject.MoleculeByName(expressionProfile.MoleculeName);
+         // SimulationSubject To ExpressionProfile. We do not update the parameter origin id in the target entities (expression profile)
+         synchronizeExpressionProfiles(moleculeInIndividual, simulationSubject, expressionProfile.Molecule, expressionProfile.Individual, updateParameterOriginId: false);
+
+         //however we need to make sure that we reference the expression profile parameter in the source individual
+
+         var allExpressionProfileParameters = allMoleculeParametersFor(expressionProfile.Individual, expressionProfile.Molecule);
```

Comment:
This is required for project conversion so that the parameter in individual also points to the expression profile parameter